### PR TITLE
Implement DPOS network hub feature

### PR DESCRIPTION
**Pull request by alexpan0610**

### CODE REVIEW

File: dpos/network.go [hunk 1]
```diff
@@ -313,9 +313,8 @@ func NewDposNetwork(account account.Account,
 	copy(pid[:], account.PublicKeyBytes())
 	server, err := p2p.NewServer(&p2p.Config{
 		PID:              pid,
+		EnableHub:        true,
```
Comment (bocheng0000) [inline dpos/network.go]: If this parameter is configurable, it should be taken from the config.json file; if not, it should not appear in the config.

Comment (alexpan0610) [reply dpos/network.go]: This is not the config of the config file, this config is the parameters used to initiate a P2P server instance.

File: dpos/p2p/server.go [hunk 2]
```diff
@@ -811,9 +823,16 @@ func NewServer(origCfg *Config) (*server, error) {
 		return nil, err
 	}
 
+	admgr := addrmgr.New(cfg.DataDir)
+	var hubService *hub.Hub
+	if cfg.EnableHub {
```
Comment (bocheng0000) [inline dpos/p2p/server.go]: If EnableHub is not configurable, there should be no conditional statement here.

Comment (alexpan0610) [reply dpos/p2p/server.go]: This config is designed to use in code level, for DPOS package is not only used by Elastos.ELA and also Elastos.Aribter or other projects. Elastos.ELA DPOS network provides hub service for other services is a feature and must enabled, but Elastos.Aribter or other projects can decide enable or disable hub service by their needs.

**Pull request merged**